tracpro/msgs/views.py

```diff
@@ -170,8 +169,8 @@ def post(self, request, *args, **kwargs):
                 # Wait 1 second to allow message to reach server then run the task
```

Comment:
This comment about waiting 1 second doesn't appear to apply anymore.  Or maybe the time.sleep(1) needs to be put back in?